doc/changes/unreleased.md

```diff
@@ -13,6 +13,10 @@
 
 * #33: Updated to Python 3.10
 
+## Documentation
+
+* Update project documentation
```

Comment:
We usually add a reference to the ticket here and also strive to use past tense.
Additionally, "Update project documentation" below the heading "Documentation" doesn't give to much additional information about the actual changes.

Proposal
```suggestion
* #<ticket>: Added section .... to user guide
```

Reply:
Well there is no ticket for this change ;P